### ISSUE TITLE
Feature request: library for developers to integrate with steam that asks for credentials

### ISSUE THREAD

**Issue by Wallby** [ValveSoftware/steam-for-linux#8257]
Ever considered to make a library such that before starting a game the user is prompted for their credentials..? (similar to how git asks for credentials when you run it unless you configured the credentials to be saved).
This way you still have the developer protection (i.e. they must have an account to play the game) but you don't require to download the steam client (i.e. just download the game via your webbrowser <- starting is not possible without entering credentials anyway).

I think it doesn't make any sense that you have to download an application to download games, as a webbrowser is perfectly capable of downloading games. To e.g. unlock an achievement.. the library could provide a function to communicate with a web API and specify the credentials (identical to what probably happens right now).

**Comment by kisak-valve** [comment]
Hello @Wallby, https://steamcommunity.com/discussions/forum/10/ is a better place to discuss this kind of OS-independent feature request.

From the Steam for Linux client perspective, this request is out of scope for the issue tracker
